src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala

```diff
@@ -0,0 +1,13 @@
+package jp.ne.opt.chronoscala
+
+import jp.ne.opt.chronoscala.Tag.CS
+
+import scala.language.implicitConversions
+
+trait NamespacedImplicits extends Implicits with NamespacedIntImplicits
+
+trait NamespacedIntImplicits extends IntImplicits {
+  override def richInt(n: Int): RichInt = new RichInt(n)
```

Comment:
I think disabling Int conversion by overriding is little bit tricky.

How about following ?

- `NamespacedIntImplicits` extends nothing.
- `NamespacedImplicits extends NamespacedIntImplicits with DurationImplicits with TimeImplicits with OrderingImplicits`
- `NamespacedImports extends TypeImports with StaticForwarderImports with NamespacedImplicits`